app/scripts/modules/core/src/presentation/forms/fields/formikField.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { IFieldProps, IFormikFieldProps, Omit } from '../interface';
+
+export type IFormikField<P extends IFieldProps<T>, T = string> = React.ComponentType<
```

Comment:
This is a type adapter for a FormField which allows it to use Formik props.  It removes `value` and `onChange` from the allowable props and adds `formik` and `name`.